lib/bugsnag/resque.rb

```diff
@@ -37,4 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Bugsnag.configuration.app_type = "resque"
+Resque.before_first_fork do
+  Bugsnag.configuration.app_type = "resque"
+  Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
Why is this setting synchronous? How does we override this value in resque workers if we don't want it?